### PR TITLE
Some fixes and translation updates

### PR DESCRIPTION
**Pull request by Dushess**
Mostly for RU section, but also touched some DE data.

### CODE REVIEW

**Comment by Dushess** [comment]
Oh, need to swap "Расшифровка" to "Усл.открытия"

File: EDEngineer/Resources/Data/localization.json [hunk 1]
```diff
@@ -9909,7 +9909,7 @@
       "pt": null,
       "de": null,
       "fr": null,
-      "ru": null,
+      "ru": "Урон в голову (плазменное от Manticore),
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:9912]: missing double quote here

File: EDEngineer/Resources/Data/localization.json [hunk 2]
```diff
@@ -9921,7 +9921,7 @@
       "pt": null,
       "de": null,
       "fr": null,
-      "ru": null,
+      "ru": "Урон в голову (пулевое от Kinematic),
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:9924]: missing double quote here

File: EDEngineer/Resources/Data/localization.json [hunk 3]
```diff
@@ -9945,7 +9945,7 @@
       "pt": null,
       "de": null,
       "fr": null,
-      "ru": null,
+      "ru": "Урон в голову (лазерное от Takada),
```
Comment (msarilar) [inline EDEngineer/Resources/Data/localization.json:9948]: missing double quote here

**Comment by msarilar** [comment]
thanks, nice work! only a few missing double quotes before I can merge

**Review by msarilar — APPROVED** [review]
great stuff thanks a lot!

**Comment by Dushess** [comment]
Just now added some Deutsch changes, you can take it in my fork's file.

**Comment by msarilar** [comment]
if you have more changes you should open a second pull request

**Comment by Dushess** [comment]
Wait, how to select file in my fork as source?

**Pull request merged**